items/JUNGLE_HEART.json

```diff
@@ -1,10 +1,9 @@
 {
   "itemid": "minecraft:skull",
   "displayname": "§6Jungle Heart",
-  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"1a136205-4ff3-3028-bb5b-c2ef8a047399\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTYxODIzMzEwNTIyOSwKICAicHJvZmlsZUlkIiA6ICIzYTNmNzhkZmExZjQ0OTllYjE5NjlmYzlkOTEwZGYwYyIsCiAgInByb2ZpbGVOYW1lIiA6ICJOb19jcmVyYXIiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNzRjZjNjNWYyNGM5YmY4YjkwMzFlOGFlZDUzMzdiNmEwN2Y4MTljZGU4YzYyMjA4NTU0NTE2MmY4YTY5MzA2MCIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9\"}]}},display:{Lore:[0:\"\",1:\"§7§7Found in the deepest layers of\",2:\"§7the Jungle Temple\",3:\"\",4:\"§6§lLEGENDARY\"],Name:\"§6Jungle Heart\"},ExtraAttributes:{originTag:\"UNKNOWN\",id:\"JUNGLE_HEART\",uuid:\"3cceb75c-737a-4446-be6d-0c398c4f0cc9\",timestamp:\"7/16/21 1:04 PM\"}}",
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"1a136205-4ff3-3028-bb5b-c2ef8a047399\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTYxODIzMzEwNTIyOSwKICAicHJvZmlsZUlkIiA6ICIzYTNmNzhkZmExZjQ0OTllYjE5NjlmYzlkOTEwZGYwYyIsCiAgInByb2ZpbGVOYW1lIiA6ICJOb19jcmVyYXIiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNzRjZjNjNWYyNGM5YmY4YjkwMzFlOGFlZDUzMzdiNmEwN2Y4MTljZGU4YzYyMjA4NTU0NTE2MmY4YTY5MzA2MCIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9\"}]}},display:{Lore:[0:\"§7§7Found in the deepest layers of\",1:\"§7the Jungle Temple\",2:\"\",3:\"§6§lLEGENDARY\"],Name:\"§6Jungle Heart\"},ExtraAttributes:{originTag:\"UNKNOWN\",id:\"JUNGLE_HEART\",uuid:\"3cceb75c-737a-4446-be6d-0c398c4f0cc9\",timestamp:\"7/16/21 1:04 PM\"}}",
```

Comment:
While you are there could you remove the timestamp and uuid at the end of the nbt tag. Idk while its still there